### PR TITLE
tikv-control.md: update tikv-control.md for raft region

### PR DESCRIPTION
**Pull request by HuSharp**
Signed-off-by: husharp <jinhao.hu@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): ref https://github.com/tikv/tikv/issues/13760 & Pr is merged https://github.com/tikv/tikv/pull/13768

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

**Comment by JmPotato** [comment]
@TomShawn PTAL, thx.

File: tikv-control.md [hunk 1]
```diff
@@ -153,22 +153,65 @@ AAFF
 
 `raft` 子命令可以查看 Raft 状态机在某一时刻的状态。状态信息包括 **RegionLocalState**、**RaftLocalState** 和 **RegionApplyState** 三个结构体，及某一条 log 对应的 Entries。
 
-你可以使用 `region` 和 `log` 两个子命令分别查询以上信息。两条子命令都同时支持远程模式和本地模式。其用法及输出内容如下所示：
+可以使用 `region` 和 `log` 两个子命令分别查询以上信息。两条子命令都同时支持远程模式和本地模式。
+
+对于 `region` 命令：
```
Comment (Oreoxmt) [inline tikv-control.md:158]:
```suggestion
对于 `region` 子命令：
```

**Comment by HuSharp** [comment]
Thx for ur advice, Address all comments, PTAL! @Oreoxmt

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: tikv-control.md [hunk 2]
```diff
@@ -153,30 +147,67 @@ AAFF
 
 `raft` 子命令可以查看 Raft 状态机在某一时刻的状态。状态信息包括 **RegionLocalState**、**RaftLocalState** 和 **RegionApplyState** 三个结构体，及某一条 log 对应的 Entries。
 
-你可以使用 `region` 和 `log` 两个子命令分别查询以上信息。两条子命令都同时支持远程模式和本地模式。其用法及输出内容如下所示：
+可以使用 `region` 和 `log` 两个子命令分别查询以上信息。两条子命令都同时支持远程模式和本地模式。
+
+对于 `region` 子命令：
+
+- 要查看指定的 Region，可在命令中使用 `-r` 参数，多个 Region 以 `,` 分隔。也可以使用 `--all-regions` 参数来返回所有 Region（`-r` 与 `--all-regions` 不能同时使用）
+- 要限制打印的 Region 的数量，可在命令中使用 `--limit` 参数（默认为 `16`）
```
Comment (TomShawn) [inline tikv-control.md:155]:
```suggestion
- 要限制输出的 Region 的数量，可在命令中使用 `--limit` 参数（默认为 `16`）
```

File: tikv-control.md [hunk 3]
```diff
@@ -153,30 +147,67 @@ AAFF
...
+- 要查询某个 key 范围中包含哪些 Region，可在命令中使用 `--start` 和 `--end` 参数（默认不限范围，采用 Hex 格式）
 
-{{< copyable "shell-regular" >}}
+需要打印 id 为 `1239` 的 Region 时，用法及输出内容如下所示：
```
Comment (TomShawn) [inline tikv-control.md:158]:
```suggestion
需要输出 id 为 `1239` 的 Region 时，用法及输出内容如下所示：
```

File: tikv-control.md [hunk 4]
```diff
@@ -153,30 +147,67 @@ AAFF
...
+}
+"apply state": {
+    applied_index: 8 commit_index: 8 commit_term: 8
+    truncated_state {index: 5 term: 5} 
+}
+```
+
+需要查询某个 key 范围中包含哪些 Region 时，用法及输出内容如下所示：
+
+- 当 key 范围位于某个 Region 中时，将会输出该 Region 信息。
```
Comment (TomShawn) [inline tikv-control.md:185]:
```suggestion
- 当 key 范围包含在某个 Region 中时，将会输出该 Region 信息。
```

File: tikv-control.md [hunk 5]
```diff
@@ -153,30 +147,67 @@ AAFF
...
+- 当 key 范围精准到某个 Region 时，以上述 Region `1239` 为例：当给定的 key 范围为 Region `1239` 的范围时，由于 Region 范围为左闭右开区间，并且 Region `1009` 以 Region `1239` 的 `end_key` 作为 `start_key`，因此会同时输出 Region `1009` 和 Region `1239` 的信息。
```
Comment (TomShawn) [inline tikv-control.md:186]:
```suggestion
- 当 key 范围精准到某个 Region 的范围时，以上述 Region `1239` 为例：当给定的 key 范围为 Region `1239` 的范围时，由于 Region 范围为左闭右开区间，并且 Region `1009` 以 Region `1239` 的 `end_key` 作为 `start_key`，因此会同时输出 Region `1009` 和 Region `1239` 的信息。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**